recipes/orange3/meta.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: orange3
+  version: 3.3.6
+
+source:
+  fn: Orange3-3.3.6.tar.gz
+  url: https://github.com/biolab/orange3/archive/3.3.6.tar.gz
```

Comment:
Please change to `url: https://github.com/biolab/orange3/archive/{{ version }}.tar.gz`.